Aspire Budgeting/Facilities/AppCoordinator.swift

```diff
@@ -0,0 +1,44 @@
+//
+// AppCoordinator.swift
+// Aspire Budgeting
+//
+
+import Foundation
+import Combine
+
+class AppCoordinator: ObservableObject {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/f16e8388717510c8ad38f72d1539d0d9d847baf4/Aspire Budgeting/Facilities/AppCoordinator.swift#L9"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>